### PR TITLE
Old person picture command

### PR DESCRIPTION
**Pull request by ghost**
- Install axios for HTTP requests.
- FIX: Added ability for ResponseManager to correctly handle promises.
- Integrated with Azure Bing Image search API to return pictures of old people on '$pic' command.
- Added associated unit test for new command.

### CODE REVIEW

**Comment by ssp0929** [comment]
Will need to add another attribute to the JSON object in the auth.json file for the Azure token.

File: src/responses/commands.ts [hunk 1]
```diff
@@ -48,6 +49,32 @@ export const uptime = new Response(
   "I tell you how long I've been awake :timer:"
 )
 
+const picPhrase = RegExp(`${CMD_PREFIX}.*`)
```
Comment (cargillj) [inline src/responses/commands.ts]: Shouldn't you be searching for $pic here? This regex matches on all commands.

File: src/responses/commands.ts [hunk 2]
```diff
@@ -54,20 +55,18 @@ export const pic = new Response(
...
       headers: {
-        Authorization: 'Client-ID xxxxxxxxxxREDACTED'
+        "Ocp-Apim-Subscription-Key": auth.bingToken
       }
     });
 
-    if (search.status === 200 && search.data.data.length) {
-      const resultArray = search.data.data;
-      return resultArray[Math.floor(Math.random() * Math.floor(resultArray.length))].link;
+    console.log(search);
```
Comment (cargillj) [inline src/responses/commands.ts]: nit. remove or use logger here.

**Review by cargillj — CHANGES_REQUESTED** [review]

File: src/responses/commands.ts (hunk 2 above)
Comment (ssp0929) [reply src/responses/commands.ts]: Oops.

File: src/responses/commands.ts (hunk 1 above)
Comment (ssp0929) [reply src/responses/commands.ts]: Oops.

**Review by cargillj — APPROVED** [review]
Interesting that using wildcard here didn't mess cause tests to fail. We may want to fix tests so they fail with regex collisions.

**Pull request merged**